config/spryk/spryks/Spryker/AddModule.yml

```diff
@@ -1,5 +1,5 @@
 spryk: structure
-description: "Adds a new *module* structure. Creates only empty skeleton without any Yves/Zed/Glue classes. Preferable to use AddDefaultModule"
+description: "Adds a new *module* structure. Creates only empty skeleton without any Yves/Zed/Glue classes. Preferable to use AddModuleDefault"
```

Comment:
after all renaming please also check that `spryk-gui` is still working from the Backoffice `/spryk-gui/list`